ddtrace/internal/encoding.py

```diff
@@ -50,12 +50,51 @@ def encode(self, obj):
         """
         raise NotImplementedError()
 
+    @staticmethod
+    def _span_to_dict(span):
```

Comment:
We probably want to hold off making the change here because we'll need to do it in 1.0/1.x.

Reply:
This is a good point, we only need to deprecate in this PR, the removal + internal move can happen on 1.x

Reply:
That makes sense. Will pull that out.